.github/workflows/pages.yml

```
@@ -61,40 +48,44 @@ jobs:
-
-  # Deployment job for other branches (without environment protection)
-  deploy-dev:
-    name: Deploy to Development
-    if: github.ref != 'refs/heads/master' && !(github.event_name == 'workflow_dispatch' && github.event.inputs.branch == 'master')
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v4
+deploy:
```

actionlint (1.7.7)

unexpected key "deploy" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)